chainer/functions/connection/deconvolution_2d.py

```diff
@@ -325,21 +325,49 @@ def deconvolution_2d(x, W, b=None, stride=1, pad=0,
                      outsize=None, use_cudnn=True, deterministic=False):
     """Two dimensional deconvolution function.
 
-    This is an implementation of two-dimensional deconvolution.
+    This is an implementation of two-dimensional deconvolution (alse known as
+    transposed convolution).
```

Comment:
Yes, we cannot do that in at least `v1`. But it is confusing for users if we write transposed convolution is an alias of deconvolution. So, it is better to write explicitly that in most of deep learning frameworks and papers, this is called transposed convolution but for historical reasons,  (e.g. paper by Ziller [here](http://www.matthewzeiler.com/pubs/cvpr2010/cvpr2010.pdf)) we call it deconvolution.